File: CB.EN.U4CYS22057/30_03_2023/python/frequency_in_an_array.py
# Declaring array
array = [0] * (10)

# Declaring other variables
# for loop for receiving inputs
for x in range(0, 9 + 1, 1):
    print("Enter " + str(x + 1) + " value")
    input = int(input())
    
    # assigning values in the array
    array[x] = input

# Getting the value to check the frequency
print("Enter the value for which the frequency should be found")
value = int(input())
freq = 0

# For loop for checking frequnency
for x in range(0, 9 + 1, 1):
    if array[x] == value:
        freq = freq + 1

# if condition to show an output for values not present in the array
if freq == 0:
    print("The given input value does not exist in the array")
else:
    
    # output for correct values
    print("The frequency of the given value in array is " + str(freq))

File: CB.EN.U4CYS22057/30_03_2023/c/frequency_in_an_array.c
#include <stdio.h>

int main() {
    // Declaring array
    int array[10] = {0};

    // For loop for receiving inputs
    for (int x = 0; x < 10; x++) {
        printf("Enter %d value:\n", x+1);
        int input;
        scanf("%d", &input);

        // assigning values in the array
        array[x] = input;
    }

    // Getting the value to check the frequency
    printf("Enter the value for which the frequency should be found:\n");
    int value;
    scanf("%d", &value);
    int freq = 0;

    // For loop for checking frequency
    for (int x = 0; x < 10; x++) {
        if (array[x] == value) {
            freq++;
        }
    }

    // If condition to show an output for values not present in the array
    if (freq == 0) {
        printf("The given input value does not exist in the array\n");
    } else {
        // Output for correct values
        printf("The frequency of the given value in array is %d\n", freq);
    }

    return 0;
}
